internal/perf/manager/manager.go

```diff
@@ -79,14 +79,17 @@ func New(configLoader config.Loader, router metrics.Router, opts ...Option) (*ma
 
 	perfEvents := channel.NewMerger[performance.Event]()
 
+	// Get host paths from environment
+	hostPaths := environment.GetHostPaths()
+
 	m := &manager{
 		configLoader:      configLoader,
 		router:            router,
 		perfEvents:        perfEvents,
 		runningCollectors: make(map[string]*collectorInstance),
-		procPath:          os.Getenv("HOST_PROC"),
-		sysPath:           os.Getenv("HOST_SYS"),
-		devPath:           os.Getenv("HOST_DEV"),
+		procPath:          hostPaths.Proc,
+		sysPath:           hostPaths.Sys,
+		devPath:           hostPaths.Dev,
```

Comment:
note that these also get populated via `collectionConfig.ApplyDefaults()`. I think the collector config and general API could still use of some iterations but I like that it is more explicit here.